packages/ui/src/views/repo/pull-request/details/pull-request-conversation-page.tsx

```diff
@@ -30,7 +33,8 @@ export const PullRequestConversationPage: FC<PullRequestConversationPageProps> =
   filtersProps,
   overviewProps,
   commentBoxProps,
-  sideBarProps
+  sideBarProps,
+  principalProps
```

Comment:
`principal` props are being passed from `gitness` to display and search the users list.